pkg/provisioningv2/rke2/planner/etcdrestore.go

```diff
@@ -129,126 +121,132 @@ func (e *etcdRestore) restorePlan(controlPlane *rkev1.RKEControlPlane, snapshot
 		}),
 	}
 
-	return commonNodePlan(e.secrets, controlPlane, nodePlan)
+	return nodePlan, nil
 }
 
-func (e *etcdRestore) stopControlPlanePlan(controlPlane *rkev1.RKEControlPlane) (plan.NodePlan, error) {
-	nodePlan := plan.NodePlan{
-		Instructions: []plan.Instruction{
-			ensureInstalledInstruction(controlPlane),
```

Comment:
This can be removed because it is replaced with https://github.com/rancher/rancher/pull/36026/files/6b8159b055837a3d1b28471270865ecf74901182#diff-5f92b3ab69d35a428cbc5f551fd3cca0264bdbef1d05e4d0f0f4a521e6ef775dR133 (I think that link works), it's the `p.generateInstallInstructionWithSkipStart`